README.md

```diff
@@ -66,6 +66,7 @@ PRs are welcome. Please visit [#174](https://github.com/jarun/Buku/issues/174) f
 - Portable, merge-able database to sync between systems
 - Multithreaded full DB refresh
 - Shell completion scripts, man page with handy examples
+- Colored output
```

Comment:
Please move it to the top point:

`Lightweight, clean interface, custom colors`

And add this in the man page too (we show the features in man page as well)